lib/tracker/mode_s/frame.go

```diff
@@ -304,6 +305,8 @@ type (
 		alert          bool
 		// if we have trouble decoding our frame, the message ends up here
 		err error
+
+		decodeLock sync.Mutex
```

Comment:
Does this get used anywhere?

Reply:
yes, it is the  thing that stops 2 decodes at the same time